src/main/java/org/beanpod/switchboard/config/WebMvcConfiguration.java

```diff
@@ -12,7 +12,10 @@ public WebMvcConfigurer webMvcConfigurer() {
     return new WebMvcConfigurer() {
       @Override
       public void addCorsMappings(CorsRegistry registry) {
-        registry.addMapping("/**");
+        registry
+            .addMapping("/**")
+            .allowedOrigins("*")
```

Comment:
This line is causing a security hotspot. It's too permissive. Do you want to fix by explicitly defining allowed origins? Or, won't fix?

Reply:
Fix still works without the `allowedOrigins` line so i removed it, the security hotspot should be gone